src/index.html

```diff
@@ -7,6 +7,289 @@
+            <span class="service-card__title">Architecture</span>
+          </div>
+
+          <div class="service-card card-blue">
+            <img src="./assets/icons/electric.svg" alt="service icon" class="service-card__icon">
+            <div class="service-card__separator"></div>
+            <span class="service-card__title">Electric</span>
+          </div>
+
+        </div>
+      </section>
+<!-- ------------------ EXPERIENCE --------------------- -->
+      <section class="experience">
+        <div class="experience__cards">
+
+          <div class="experience-card card-right">
+            <div class="experience-card__icon">
+              <img src="./assets/icons/clients.svg" alt="clients icon">
+            </div>
+            <span class="experience_card__digit">84</span>
+            <span class="experience-card__description">Happy Clients</span>
+          </div>
+
+          <div class="experience-card card-left">
+            <div class="experience-card__icon">
+              <img src="./assets/icons/projects.svg" alt="projects icon">
+            </div>
+            <span class="experience_card__digit">123</span>
+            <span class="experience-card__description">Projects Completed</span>
+          </div>
+
+          <div class="experience-card card-right">
+            <div class="experience-card__icon">
+              <img src="./assets/icons/awards.svg" alt="awards icon">
+            </div>
+            <span class="experience_card__digit">37</span>
+            <span class="experience-card__description">Awards Win</span>
+          </div>
+
+          <div class="experience-card card-bottom">
+            <div class="experience-card__icon">
+              <img src="./assets/icons/business.svg" alt="business icon">
+            </div>
+            <span class="experience_card__digit">30</span>
+            <span class="experience-card__description">Years in Business</span>
+          </div>
+
+        </div>
+        <div class="experience__description">
+          <span class="experience__description__title">30 Years Experience</span>
+          <p class="experience__description__content">Our company has been the leading provided construction services to clients throughout the USA since 1988.</p>
+          <button href="#" class="experience__description__btn">Contact Us</button>
+        </div>
+      </section>
+<!-- ------------------ CONSULTATION --------------------- -->
+      <section class="consultation">
+        <img src="./assets/images/room-background.png" alt="room background photo" class="sonsultation__img">
+        <div class="consultation__info">
+          <span class="consultation__info-descr">Free consultation with exceptional quality</span>
+          <span class="consultation__info-tel">Just one call away: <a href="tel:+84 1102 2703">+84 1102 2703</a></span>
+        </div>
+        <button href="#" class="consultation__btn">Get your consultation</button>
```

Comment:
`btn btn--white btn--outline`

Reply:
<img width="190" alt="Screenshot 2023-10-26 at 16 00 40" src="https://github.com/trofimovama/thebox-landing/assets/98819201/27600af1-b42d-4daf-ba05-f1d4a86ed912">

<img width="202" alt="Screenshot 2023-10-26 at 16 01 51" src="https://github.com/trofimovama/thebox-landing/assets/98819201/321059b5-1abc-427b-9cd0-1fdcc44ec813">

.btn--white was used for solid white. I added .btn--outline to make bg transparent in this class